packages/components/src/elevation/types.ts

```diff
@@ -35,18 +39,22 @@ export type Props = {
 	 */
 	offset?: number;
 	/**
-	 * Size of the shadow, based on the Style system's elevation system. The `value` determines the strength of the shadow, which sense of depth.
-	 * In the example below, `isInteractive` is activated to give a better sense of depth.
+	 * Size of the shadow, based on the Style system's elevation system. The
+	 * `value` determines the strength of the shadow, which sense of depth.
+	 *
+	 * In the example below, `isInteractive` is activated to give a better sense
+	 * of depth.
 	 *
 	 * @example
 	 * ```jsx
 	 * import { __experimentalElevation as Elevation } from '@wordpress/components';
-	 * 	* function Example() {
-	 * 	return (
-	 * 		<div>
-	 * 			<Elevation isInteractive value={ 200 } />
-	 * 		</div>
-	 * 	);
+	 *
+	 * function Example() {
```

Comment:
Personally, this note and example about the `isInteractive` doesn't quite make sense to me here as part of the `value` prop description, and made me more confused 😅 What do you think about removing them? The description for the `isInteractive` prop itself, coupled with the improved Storybook stories, should be sufficient.

Reply:
Yes I agree. Removed. 👍